maintainers/scripts/debian-patches.sh

```diff
@@ -1,17 +1,17 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
The shell script uses features outside of POSIX sh. Specifically, these are bash features.